tests/test_merge.py

```diff
@@ -0,0 +1,34 @@
+from geopandas._compat import import_optional_dependency
+
+from fiboa_cli import merge, validate
+from click.testing import CliRunner
+from glob import glob
+
+"""
+Create input files with:
+
+$ for i in at be_wa nl; do fiboa convert $i -o tests/data-files/merge/$i.parquet -c tests/data-files/convert/$i; done
+"""
+
+
+def test_converter(tmp_file):
+    # merge files in directory
+    files = glob("tests/data-files/merge/*.parquet")
+    runner = CliRunner()
+    args = files + ['-o', tmp_file.name]
+    result = runner.invoke(merge, args)
+    assert result.exit_code == 0, result.output
+
+    # Merged parquet file should be valid
+    result = runner.invoke(validate, [tmp_file.name, '--data'])
+    assert result.exit_code == 0, result.output
+
+    parquet = import_optional_dependency(
+        "pyarrow.parquet", extra="pyarrow is required for Parquet support."
+    )
```

Comment:
Is that an actual issue? We use it in other places, too, so it's not really optional, isn't it?

Reply:
I searched in the code on how to open/read/write parquet files (for testing this case). This is what I found in `geopandas.py` and I bluntly assumed this is how I should import it... Feel free to change